<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>580</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1025</width>
    <height>560</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Arial Black</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>CDM_Lab6_Graphs</string>
  </property>
  <property name="windowIcon">
   <iconset resource="images.qrc">
    <normaloff>:/gc_logo.png</normaloff>:/gc_logo.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">*{
	font-family: 'Arial Black', Arial, sans-serif;
	background-color: #242b35;/**/
}
QTableWidget{
	background-color:white;
}
QSpinBox{
	background-color: white;
	font-size:12pt;
}
QLabel{
	color:white;/**/
	font-size:12pt;
}
QPushButton {
	border:none;
	background-color: rgba(109, 135, 214, 0.4);
	color:white;/**/
	font-size:12pt;
	border-radius:10px;
}
QPushButton:hover {
	background-color:#FF7800	;/**/
	color:white;/**/
}
QPushButton:pressed {
	background-color:#3E13AF;/**/
	color:white;/**/
}
QGroupBox{
	border:3px solid #4c495c;/**/
	color:white;/**/
	font-size:12pt;
}</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="font">
    <font>
     <family>Arial Black</family>
    </font>
   </property>
   <property name="toolTipDuration">
    <number>-1</number>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QFrame" name="frame">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>561</width>
        <height>561</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>561</width>
        <height>561</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="title">
         <string/>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <item>
            <widget class="QLabel" name="VERTICE">
             <property name="text">
              <string>Amount of vertice:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="vertice">
             <property name="minimumSize">
              <size>
               <width>70</width>
               <height>32</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>70</width>
               <height>35</height>
              </size>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_7">
           <item>
            <widget class="QLabel" name="startVer">
             <property name="text">
              <string>Start vertice:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="start">
             <property name="minimumSize">
              <size>
               <width>70</width>
               <height>32</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>70</width>
               <height>35</height>
              </size>
             </property>
             <property name="minimum">
              <number>1</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="endVer">
             <property name="text">
              <string>End Vertice</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="end">
             <property name="maximumSize">
              <size>
               <width>70</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="minimum">
              <number>1</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <property name="topMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QPushButton" name="find">
             <property name="minimumSize">
              <size>
               <width>100</width>
               <height>32</height>
              </size>
             </property>
             <property name="text">
              <string>Find</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="result">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>30</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>Arial Black</family>
              </font>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <widget class="QPushButton" name="matrix">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>30</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="text">
              <string>ToMatrix</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="build">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>30</height>
              </size>
             </property>
             <property name="text">
              <string>ToGraph</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="clear">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>30</height>
              </size>
             </property>
             <property name="text">
              <string>Clear</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>250</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="title">
         <string/>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QLabel" name="aboveFirstMatrix">
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>20</height>
              </size>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QTableWidget" name="firstMatrix">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>0</width>
               <height>0</height>
              </size>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="styleSheet">
         <string notr="true">margin-left:10px;</string>
        </property>
        <property name="text">
         <string>Laboratory work â„–6 &quot;Finding minimal way &quot; was made by
Konovalenko Svitlana, Krasenko Artemiy, Potapenko Rodion</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QPushButton" name="exit">
          <property name="minimumSize">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color:red;</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_4">
          <property name="minimumSize">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color:#242b35;</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_5">
          <property name="minimumSize">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color:#242b35;</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QFrame" name="switch_2">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>10</width>
            <height>46</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>20</width>
            <height>46</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">border-radius:10px;
background-color:yellow;</string>
          </property>
          <property name="lineWidth">
           <number>0</number>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_6">
           <property name="spacing">
            <number>6</number>
           </property>
           <property name="sizeConstraint">
            <enum>QLayout::SetMinAndMaxSize</enum>
           </property>
           <property name="leftMargin">
            <number>1</number>
           </property>
           <property name="topMargin">
            <number>1</number>
           </property>
           <property name="rightMargin">
            <number>1</number>
           </property>
           <property name="bottomMargin">
            <number>1</number>
           </property>
           <item>
            <widget class="QPushButton" name="day">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>18</width>
               <height>18</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>18</width>
               <height>18</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color:white;
border-radius:9px;
margin:0px;</string>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="night">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>18</width>
               <height>18</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>18</width>
               <height>18</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: yellow;/*#242b35;*/
border-radius:9px;
margin:0px;</string>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
